pkg/GAPJulia/JuliaInterface/julia/ccalls.jl

```diff
@@ -125,10 +125,22 @@ end
 > There is no argument number checking here, all checks on the arguments
 > are done by GAP itself.
 """
-(func::MPtr)(args...) = call_gap_func(func,args...)
-
-function call_gap_func(func::MPtr,args...)
-    return ccall(:call_gap_func, Any, (MPtr, Any), func, args)
+(func::MPtr)(args...; kwargs...) = call_gap_func(func,args...; kwargs...)
```

Comment:
Please remove my other commit from this PR.

Reply:
OK.